client/src/tests/MovieContainer.test.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import renderer from 'react-test-renderer';
+import { mount } from 'enzyme';
+import MovieContainer from '../components/MoviesContainer';
+import movies from '../components/movies';
+
+it ('renders correctly', ()=>{
+    const component = renderer.create(<MovieContainer />);
+
+    var jsonComponent = component.toJSON();
+    expect(jsonComponent).toMatchSnapshot();
+});
+
+it ('loads movie details when some movie is selected', ()=>{
+    const component = mount(<MovieContainer />);
+    const movieslist = component.find('.movie')
+    expect(component.find('.movie').length).toEqual(7);
```

Comment:
_minor_

To me, it would be better to split this unit test into multiple ones, inasmuch as there are a lot of assertions are happening here and it's difficult to keep track the purpose of this test.

I mean, for instance, you can define a separate test with this assertion like so

```javascript
it('should render movie list', () => {
  const component = mount(<MovieContainer />);
  const movieslist = component.find('.movie');

  expect(movieslist.length).toEqual(7);
});
```